waiter/src/waiter/state.clj

```diff
@@ -1320,6 +1320,7 @@
                                                   rem-instance-ids (filterv (complement curr-instance-ids) prev-instance-ids)
                                                   unhealthy-instance-ids (mapv :id (get service-id->unhealthy-instances' service-id))]
                                               (log/info "update-healthy-instances:" service-id "has"
+                                                        (count expired-instances) "expired instance(s),"
```

Comment:
Should we log the new `expired-healthy-instances` and `expired-unhealthy-instances` values?

Reply:
Done.